larq/quantized_variable.py

```diff
@@ -11,8 +10,9 @@
 from larq.quantizers import Quantizer
 
 try:  # pragma: no cover
-    from tensorflow.python.types.core import Tensor as TensorType  # type: ignore
-    from tensorflow.python.distribute.ps_values import AggregatingVariable  # type: ignore
+    # type: ignore
```

Comment:
I thought these `# type: ignore` comments applied to the current line? Does this apply to the following line?

Reply:
Good catch, this would actually disable silence type errors for the entire file. Fixed it in the last commit.